djstripe/enums.py

```diff
@@ -691,6 +691,12 @@ class SubmitTypeStatus(Enum):
     pay = _("pay")
 
 
+class SubscriptionProrationBehavior(Enum):
+    create_prorations = _("Create prorations")
+    always_invoice = _("Always invoice")
```

Comment:
where did you find `always_invoice`?

Reply:
From https://stripe.com/docs/billing/subscriptions/upgrade-downgrade#immediate-payment

> If you want the customer to immediately pay the price difference when switching to a more expensive subscription on the same billing cycle, you can set proration_behavior to always_invoice, which will calculate the proration and then immediately generate an invoice after making the switch. You can combine this behavior with pending updates so that the subscription is only updated if payment succeeds on the new invoice.